HW04/anastasia_evsihneyeva/lib/mentor.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Mentor
+  def initialize(data)
+    @name = data[:name]
+    @surname = data[:surname]
+    @students = []
+  end
+
+  def add_homework(data)
+    Notification.new(log: "-New homework was added! Name #{data[:title]}")
+    name = "#{@name} #{@surname}"
+    Homework.new(data, name)
```

Comment:
It seems to me that homework should contain information about the mentor who created it and possibly about the student to whom this homework was given.